prince-api.js

```diff
@@ -290,6 +301,10 @@ Prince.prototype.execute = function () {
     this.config.inputs.forEach(function (input) {
         args.push(input);
     });
+    this.config.cookies.forEach(function (cookie) {
+        args.push("--cookie");
+        args.push(this.config.cookies[i]);
```

Comment:
Sorry, this cannot be right. Seems like you wanted to code `args.push(this.config.cookie);`. Please test this code beforehand. It can never be worked, I think.

Reply:
You are absolutely right, it should have been just "cookie".